frontend/app/src/store/assets/index.ts

```diff
@@ -303,7 +303,129 @@ export const useAssetInfoRetrieval = defineStore(
   }
 );
 
+export const useIgnoredAssets = defineStore('ignoredAssets', () => {
```

Comment:
I know we forgot to use this scheme in some places but maybe we should start postfixing these stores with Store for clarity. 

In this case, it would be `useIgnoredAssetsStore`, it would make it easier to ell if something is a store or not wouldn't it?

Reply:
Yeah it makes sense